js/apps/admin-ui/src/i18n/i18n.ts

```diff
@@ -19,9 +21,19 @@ export const i18n = createInstance({
     escapeValue: false,
   },
   backend: {
-    loadPath: joinPath(environment.resourceUrl, `locales/{{lng}}/{{ns}}.json`),
+    loadPath: joinPath(
```

Comment:
Couple of notes here:

1. This can be made ype-safe by using [the types]([https://github.com/i18next/i18next-http-backend#typescript) of the HTTP back-end.
2. The `KeyValue` type can be directly assigned to the result of `JSON.parse`.
3. Perhaps we can use `Object.fromEntries()` to build the result? Either way this approach is also fine by me.

For example:

```ts
createInstance({
  backend: {
    parse(data) {
      const messages: KeyValue[] = JSON.parse(data);

      return Object.fromEntries(
        messages.map(({ key, value }) => [key, value] as const),
      );
    },
  },
} satisfies InitOptions<HttpBackendOptions>);
```

Reply:
That is pretty useless don't you think

Reply:
No, I don't. It's just a small enhancement, do what you see fit.